project.clj

```diff
@@ -71,4 +72,5 @@
           :ring {:nrepl {:start? true}}}]
       :test
       {:resource-paths ["test/resources"]}}
-  :repl-options {:init (set! *print-length* 10)})
+  :repl-options {:init (set! *print-length* 10)}
+  :repositories [["tmp" "file:///tmp/clojure_jars"]])
```

Comment:
Can you explain what this repository in /tmp is used for? (Don't think /tmp is the best place - won't survive reboot!)